chitterAPI.test.js

```diff
@@ -0,0 +1,51 @@
+require('jest-fetch-mock').enableMocks();
+const ChitterAPI = require('./chitterAPI');
+
+let api;
+
+describe('ChitterAPI', () => {
+  beforeEach(() => {
+    api = new ChitterAPI();
+  })
+
+  describe('loadPosts', () => {
+    it('fetches notes from the server', async () => {
```

Comment:
I don't think you need to use async functions in this case. I can understand why you would though